niworkflows/interfaces/bids.py

```diff
@@ -248,59 +241,77 @@ class _DerivativesDataSinkInputSpec(DynamicTraitedSpec, BaseInterfaceInputSpec):
     check_hdr = traits.Bool(True, usedefault=True, desc='fix headers of NIfTI outputs')
     compress = traits.Bool(desc="force compression (True) or uncompression (False)"
                                 " of the output file (default: same as input)")
-    desc = Str('', usedefault=True, desc='Label for description field')
-    extra_values = traits.List(Str)
-    in_file = InputMultiObject(File(exists=True), mandatory=True,
-                               desc='the object to be saved')
-    keep_dtype = traits.Bool(False, usedefault=True, desc='keep datatype suffix')
```

Comment:
Not necessary anymore, as the suffix is automatically propagated from the `source_file` unless it is overridden by setting `inputs.suffix`.